### DISCUSSION TITLE
How do you remove this program?

### DISCUSSION THREAD

**Discussion by mcfasa09** [kangyu-california/PersistentWindows#193] [Q&A] [answered]
My use for this program has ended. How am I able to remove it from my PC? Thanks!

**Comment by kangyu-california** [comment, marked as answer]
The uninstallation is trivial, just delete the unzipped dir where you run PW from.
Don't forget to delete C:\Users\[User]\AppData\Local\PersistentWindows as well

**Reply by mcfasa09** [reply]
Thank you!

For other users: If you followed the install instructions on GitHub, you'll need to also delete your scheduled task (in Task Scheduler). Otherwise the program will automatically start when you boot Windows.

**Reply by AnalyzeNFix** [reply]
Except StartPersistentWindows what else ?
